yt_dlp/extractor/cwtv.py

```diff
@@ -13,68 +13,24 @@
         'info_dict': {
-            'id': 'd848488f-f62a-40fd-af1f-6440b1821aab',
+            'id': '9149a1e1-4cb2-46d7-81b2-47d35bbd332b',
             'ext': 'mp4',
-            'title': 'Ready Or Not',
-            'description': 'Simone is concerned about changes taking place at Bringston; JR makes a decision about his future.',
+            'title': 'A Stitch in Time',
+            'description': 'City Protective Services officer Kiera Cameron is transported from 2077 to deal with eight of the world\'s deadliest terrorists.',
             'thumbnail': r're:^https?://.*\.jpe?g$',
-            'duration': 2547,
-            'timestamp': 1720519200,
+            'duration': 2632,
+            'timestamp': parse_iso8601('2025-01-15T08:00:00+00:00'),
             'uploader': 'CWTV',
-            'chapters': 'count:6',
-            'series': 'All American: Homecoming',
-            'season_number': 3,
+            'chapters': 'count:5',
+            'series': 'Continuum',
+            'season_number': 1,
             'episode_number': 1,
-            'age_limit': 0,
-            'upload_date': '20240709',
-            'season': 'Season 3',
-            'episode': 'Episode 1',
-        },
-        'params': {
-            # m3u8 download
-            'skip_download': True,
-        },
-    }, {
-        'url': 'http://cwtv.com/shows/arrow/legends-of-yesterday/?play=6b15e985-9345-4f60-baf8-56e96be57c63',
-        'info_dict': {
-            'id': '6b15e985-9345-4f60-baf8-56e96be57c63',
-            'ext': 'mp4',
-            'title': 'Legends of Yesterday',
-            'description': 'Oliver and Barry Allen take Kendra Saunders and Carter Hall to a remote location to keep them hidden from Vandal Savage while they figure out how to defeat him.',
-            'duration': 2665,
-            'series': 'Arrow',
-            'season_number': 4,
-            'season': '4',
-            'episode_number': 8,
-            'upload_date': '20151203',
-            'timestamp': 1449122100,
-        },
-        'params': {
-            # m3u8 download
-            'skip_download': True,
-        },
-        'skip': 'redirect to http://cwtv.com/shows/arrow/',
-    }, {
-        'url': 'http://www.cwseed.com/shows/whose-line-is-it-anyway/jeff-davis-4/?play=24282b12-ead2-42f2-95ad-26770c2c6088',
-        'info_dict': {
-            'id': '24282b12-ead2-42f2-95ad-26770c2c6088',
-            'ext': 'mp4',
-            'title': 'Jeff Davis 4',
-            'description': 'Jeff Davis is back to make you laugh.',
-            'duration': 1263,
-            'series': 'Whose Line Is It Anyway?',
-            'season_number': 11,
-            'episode_number': 20,
-            'upload_date': '20151006',
-            'timestamp': 1444107300,
             'age_limit': 14,
-            'uploader': 'CWTV',
-            'thumbnail': r're:^https?://.*\.jpe?g$',
-            'chapters': 'count:4',
-            'episode': 'Episode 20',
-            'season': 'Season 11',
```

Comment:
don't remove dead tests, add `skip` to the test dict with a reason why they are being skipped (e.g. "expired", "redirects to homepage", etc)

Reply:
Done.